.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,80 @@
+name: 🐛 Bug report
+title: "[Bug]: "
+description: Create a bug report to help us improve
+labels: "bug:general"
+body:
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: Tell us what should happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current behavior
+      description: Tell us what happens instead of the expected behavior.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Possible solution
+      description: Tell us how it could be fixed at your glance.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Context
+      description: >
+        How has this issue affected you? What are you trying to accomplish?
+        Providing context helps us come up with a solution that is most useful in the real world.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: >
+        Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce, if relevant.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Bash-it version
+      placeholder: "How to get: bash-it version"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: List of enabled plugins, themes and aliases
+      placeholder: "How to get: bash-it show plugins|themes|aliases (it is not a pipe)"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Bash version
+      placeholder: "How to get: bash --version"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Operating system and version
+      placeholder: "How to get: neofetch (or another command)"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "bash-it doctor output"
+      value: |
+        ```
+        # Smth here
```

Comment:
Replace this with `"How to get: bash-it doctor"`

Reply:
You can use [`suggestion`](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review) to enable me use suggested changes easily. 😄